domain/HousingApi.ts

```diff
@@ -484,3 +484,12 @@ export interface VerifyAuthRequest {
 export interface VerifyAuthResponse {
   accessToken: string;
 }
+
+export interface CreateEvidenceRequest {
+  userRequestedBy?: string;
+  documentTypes: Array<string>;
```

Comment:
convention with arrays is usually:
```
string[]
```
It's the same as what you have, just syntactic sugar - thought I'd mention because it appears a few times in the PR but not urgent to change it.